### PR TITLE
Create header and headline

### PR DESCRIPTION
**Pull request by aubinleyoung**
I have created a header and headline section to the index.html, linked with the style.css to build the {Responsive-Portfolio-Website-Aubin] mobile-first.

### CODE REVIEW

**Review by vickymarz — CHANGES_REQUESTED** [review]
Hi @aubinleyoung, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## ❌ Invalid Code Review Request

You have submitted a project, that does not have a detailed readme. Kindly use the microverse provided[ readme ](https://github.com/microverseinc/readme-template) template and just edit the sections instead of deleting it 👍🏽 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: style.css [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+.white { color: white; }
+
+.header-container {
+  display: flex;
+  width: auto;
+  height: auto;
+  align-items: center;
+  background-color: #3c3a39;
+  justify-content: space-between;
+  padding: 20px;
```
Comment (christianbiring1) [inline style.css:13]: - [ ] Kindly make sure you stick to the design as much as possible, The typos (`font`), spacing(`margin, padding`), and the color of your element should be similar to the Figma template 
`Hint: To see the styles applied to a give section on the page, just that section or text in Figma and on the right side you will have them popup` :smile: 
- [ ] Also make sure you have the social icons and that while background image on your page as is the case in Figma.
Your design :point_down: 
image
The Figma template :point_down: 
image

**Review by christianbiring1 — CHANGES_REQUESTED** [review]
Hi @aubinleyoung , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
#STATUS: REQUEST CHANGES :recycle: 
almost


## Highlights
- [ ] All linters are green :heavy_check_mark: 
- [ ] Descriptive PR `summary` and `title` :+1: 
- [ ] Good commit message :+1: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 1 above)
Comment (aubinleyoung) [reply style.css:13]: mobile
Thank you for your comment, but I don't know why you don't see my social media icons while I put them.

Comment (aubinleyoung) [reply style.css:13]: @christianbiring1

Comment (aubinleyoung) [reply style.css:13]: Hello, @christianbiring1 am trying to set my local font to the text but the linter displays errors what should I do if I want to use the same font as it's on the Figma design?

**Review by nedjwab — APPROVED** [review]
# 🟢 Project Approved! 🏆

## Hi @aubinleyoung    👋

Your project is complete! ✔️ 🏁

Your code was very professional 💯

There is nothing else to say other than... _it's time to merge it_ :shipit:
Congratulations! 🎉

 
## Highlights 🌟

- ✔️ Descriptive commmit messsages
- ✔️ Descriptive PR description
- ✔️ No linter Errors
- The code is top-notch! 💯 and have a huge potential.

Cheers and Happy coding!👏👏👏

## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.


_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by aubinleyoung** [comment]
Add header and headline section to my portfolio

**Pull request merged**